CMakeLists.txt

```diff
@@ -419,6 +420,12 @@ if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
   set(SWIFT_HOST_VARIANT_SDK "LINUX")
   set(SWIFT_PRIMARY_VARIANT_SDK_default "LINUX")
 
+  # Support dynamic relocation in binutils >= 2.26
```

Comment:
More precisely, we're supporting "relative relocation against protected symbols in binutils 2.26". Give them the benefit of the doubt that 2.27 might be fixed.